Week3/homework/6-step3.js

```diff
@@ -3,4 +3,15 @@
 const arr2d = [[1, 2], [3, 4], [5, 6]];
 const arr3d = [[[1, 2], [3, 4]], [[5, 6], [7, 8]]];
 
+const arr = [1, 2, 3]
+
 // add your solution here, or add a comment on how you would tackle this problem
+
+for (var i = 0; i < arr3d.length; i++) {
+    for (var j = 0; j < arr3d[i].length; j++) {
+        for (var k = 0; k < arr3d[j].length; k++) {
+            
+            console.log(arr3d[i][j][k]);
+        }
+    }
+}
```

Comment:
You should also mention in this solution what you would do about arrays bigger than 3 dimensions.